project.clj

```diff
@@ -10,6 +10,7 @@
                  [org.clojure/tools.reader "1.3.2"]
                  [environ "1.1.0"]
                  [io.replikativ/hitchhiker-tree "0.1.7"]
+                 [com.taoensso/timbre "4.10.0"]
```

Comment:
We should add a section about logging in the readme, so one can set up their timbre config that also covers our errors. Maybe also some hints about having more than just error messages but also error data like `{:type :backend-does-not-exist :config store-config}`.

Reply:
added documentation